### ISSUE TITLE
Errors when trying to use tankdump

### ISSUE THREAD

**Issue by jsmars** [glampert/reverse-engineering-dungeon-siege#2]
Hey! I was so happy when I found this project. Dungeons Siege is such a great game and I'm glad there still is interest in it!

However, running the tankdump gives me errors, and nothing can be dumped. I feel like I'm missing something basic, but I can't get it work. Using admin command prompt in windows 10, also tried different directories.

`tankdump.exe "D:\Spel\Dungeon Siege\Resources\Objects.dsres" "D:\Dev\Sources\Dungeon Siege\Objects" -D -P`
Gives me a wall of:
```
ERROR.: Unable to open file "D:\Dev\Sources\Dungeon Siege\Objects/art/bitmaps/armor/gauntlets/b_a_gntl_014.png" for writing! No such file or directory
ERROR.: Unable to open file "D:\Dev\Sources\Dungeon Siege\Objects/art/bitmaps/armor/gauntlets/b_a_gntl_016.png" for writing! No such file or directory
ERROR.: Unable to open file "D:\Dev\Sources\Dungeon Siege\Objects/art/bitmaps/armor/gauntlets/b_a_gntl_018.png" for writing! No such file or directory
ERROR.: Unable to open file "D:\Dev\Sources\Dungeon Siege\Objects/art/bitmaps/armor/gauntlets/b_a_gntl_020.png" for writing! No such file or directory
ERROR.: Unable to open file "D:\Dev\Sources\Dungeon Siege\Objects/art/bitmaps/armor/gauntlets/b_a_gntl_022.png" for writing! No such file or directory
....
```

And skipping the TGA tag and running :
```
tankdump.exe "D:\Spel\Dungeon Siege\Resources\Objects.dsres" "D:\Dev\Sources\Dungeon `Siege\Objects" -D
```

Gives me:
`ERROR.: Failed to extract 15744 resource files!`

However, running the single `-e` command does successfully extract a file, and I can use the other commands to view files and info.

**Comment by glampert** [comment]
Hi jsmars,

Sorry for the delay on this, I've completely missed your post! Will take a look and try to find out what's going wrong.

**Comment by masterjg** [comment]
Hi @glampert ,
same think happens to me too. Apparently it doesn't extract anything for some reason...

`C:\Users\maste\Documents\Projects\ds1-texture-upscaler\tools\tankdump.exe "C:\Users\maste\Documents\Projects\ds1-texture-upscaler\input\Terrain.dsres" "C:\Users\maste\Documents\Projects\ds1-texture-upscaler\temp\Terrain.dsres" -T -D`

```
...
ERROR.: Unable to open file "C:\Users\maste\Documents\Projects\ds1-texture-upscaler\temp\Terrain.dsres/art/bitmaps/terrain/teleport_hub/b_t_th01_whitemarble-dtl-a.tga" for writing! No such file or directory
ERROR.: Failed to extract 10707 resource files!
```

**Comment by glampert** [comment]
Hi everyone,

Once again apologies for taking so long to reply to this and investigate the issue.

The problem was a change in behavior of `_mkdir` on Windows 10. This should now be fixed with commit 033f0ef. I've added new Windows 10 executables in https://github.com/glampert/reverse-engineering-dungeon-siege/blob/master/build/Release_Win10_x64.zip that contain the fix.

Thanks!
